src/Api/Controllers/AccountsController.cs

```diff
@@ -708,5 +713,27 @@ public async Task PutTaxInfo([FromBody]TaxInfoUpdateRequestModel model)
             };
             await _paymentService.SaveTaxInfoAsync(user, taxInfo);
         }
+
+        [HttpDelete("sso/{organizationId}")]
+        public async Task DeleteSsoUser(string organizationId)
+        {
+            var userId = _userService.GetProperUserId(User);
+            if (!userId.HasValue)
+            {
+                throw new NotFoundException();
+            }
+
+            await _organizationService.DeleteSsoUserAsync(userId.Value, new Guid(organizationId));
+        }
+
+        [HttpGet("sso/user-identifier")]
+        public async Task<string> GetSsoUserIdentifier()
+        {
+            var user = await _userService.GetUserByPrincipalAsync(User);
+            var token = await _userService.GenerateSignInTokenAsync(user, TokenPurposes.LinkSso);
+            var bytes = Encoding.UTF8.GetBytes($"{user.Id},{token}");
+            var userIdentifier = Convert.ToBase64String(bytes);
```

Comment:
You'll need to remember to URL encode this when you pass it in the query string from the client.

Reply:
I wasn't, but I am now 😁